src/main/java/com/boardgo/domain/meeting/service/facade/MeetingCommandFacadeImpl.java

```diff
@@ -138,25 +133,35 @@ public void updateMeeting(
         }
     }
 
+    private void validateLimitParticipantCount(MeetingUpdateRequest updateRequest, Long meetingId) {
+        MeetingParticipantSubEntity meetingParticipantSubEntity =
+                meetingParticipantSubQueryUseCase.getByMeetingId(meetingId);
+        if (updateRequest.limitParticipant() <= 1
+                || meetingParticipantSubEntity.getParticipantCount()
+                        > updateRequest.limitParticipant()) {
+            throw new CustomIllegalArgumentException("현재 참여한 인원보다 최대 인원수가 커야합니다.");
+        }
+    }
+
     private String updateImage(
-            MultipartFile imageFile, List<Long> boardGameIdList, MeetingEntity meeting) {
-        // 1. 이미지 파일 수정 X, 보드게임 수정 X
-        // 2. 이미지 파일 수정 X, 보드게임 수정 O, thumbnail 사용자 등록 이미지인 경우
-        if (Objects.isNull(imageFile)
-                && (Objects.isNull(boardGameIdList)
-                        || (Objects.nonNull(meeting.getThumbnail())
-                                && meeting.getThumbnail().startsWith("meeting")))) {
-            return meeting.getThumbnail();
-        } else {
+            MultipartFile imageFile, List<Long> boardGameIdList, MeetingEntity meeting, MeetingUpdateRequest updateRequest) {
```

Comment:
필요한 보드게임Id 만 전달하는게 좋을듯해요!
```suggestion
            MultipartFile imageFile, Long boardGameId, MeetingEntity meeting, MeetingUpdateRequest updateRequest) {
```

Reply:
위에랑 연결되는 것 같아요!

Reply:
앗 .. 아니네요 . 그 이 부분은 어쩔 수 없이 
boardGameIdList가 Null 일 수도 있는데 
getFirst를 하면 NPE가 발생하기 때문에 
Null 체크를 두 번 해줘야 해서 이렇게 받고 있습니다.